### PR TITLE
A few cleanups

### PR DESCRIPTION
**Pull request by sivaraam**
These commits add a few cleanups that I saw when skimming through the document. A few might be obvious improvements and a few might be needless churns. Let me know about which ones are good/bad so that I could change them accordingly. Hope this helps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -585,10 +585,10 @@ And finally, let's cherry-pick the commit for bug #14:
 
 <a name="delete-stale-local-branches"></a>
 ### I want to delete local branches that were deleted upstream
-Once you merge a pull request on github, it gives you the option to delete the merged branch in your fork. If you aren't planning to keep working on the branch, it's cleaner to delete the local copies of the branch so you don't end up cluttering up your working checkout with a lot of stale branches.
+Once you merge a pull request on GitHub, it gives you the option to delete the merged branch in your fork. If you aren't planning to keep working on the branch, it's cleaner to delete the local copies of the branch so you don't end up cluttering up your working checkout with a lot of stale branches.
 
 ```sh
-$ git fetch -p
+$ git fetch -p upstream
```
Comment (RichardLitt) [inline README.md]: We should add a note saying to check that upstream is the remote you want to fetch from.

File: README.md [hunk 2]
```diff
@@ -678,20 +678,26 @@ You can also do:
 
 To delete a local branch:
 
+```sh
+(master)$ git branch -d my-branch
+```
+
+To delete a local branch that HAS NOT been merged to the current branch or an upstream:
```
Comment (RichardLitt) [inline README.md]: No need for all caps for HAS NOT

File: README.md [hunk 3]
```diff
@@ -729,7 +735,7 @@ $ git push -u origin HEAD
 
 If you want to check out the other default configs which ```git push``` can take, visit the documentation for Git at https://git-scm.com/docs/git-config#git-config-pushdefault
 
-If you do not want to change the git configuration, you can also use:
+If you do not want to change the configuration, you can also use:
```
Comment (RichardLitt) [inline README.md]: perhaps "the global configuration for Git" would be more clear?

Comment (sivaraam) [reply README.md]: I think it gives the wrong view. We didn't use the `--global` option in the previous command. So, we can't say "*global* configuration". Why do you think "... to change the configuration... " isn;t clear enough?

Regardless, reading the context a little I guess it's confusing readers a lot by using inconsistent terms. I'll try to fix the confusion in a commit. Let's see how it goes.

**Comment by sivaraam** [comment]
@RichardLitt Thanks for the detailed review.

>  Did you manually change all of the TOC links?

Nope. I used `doctoc` (version 1.3.0) as mentioned in my [commit message](https://github.com/k88hudson/git-flight-rules/pull/163/commits/6c64e0f2bf88189d34dd83e83447bd7f23cadc93). Any issues?

File: README.md [hunk 4]
```diff
@@ -585,12 +586,14 @@ And finally, let's cherry-pick the commit for bug #14:
...
 ### I want to delete local branches that were deleted upstream
-Once you merge a pull request on github, it gives you the option to delete the merged branch in your fork. If you aren't planning to keep working on the branch, it's cleaner to delete the local copies of the branch so you don't end up cluttering up your working checkout with a lot of stale branches.
+Once you merge a pull request on GitHub, it gives you the option to delete the merged branch in your fork. If you aren't planning to keep working on the branch, it's cleaner to delete the local copies of the branch so you don't end up cluttering up your working checkout with a lot of stale branches.
 
 ```sh
-$ git fetch -p
+$ git fetch -p upstream
 ```
 
+where, `upstream` is the e remote you want to fetch from.
```
Comment (RichardLitt) [inline README.md]: Spelling error?

File: README.md [hunk 5]
```diff
@@ -721,21 +730,17 @@ This will give you a local copy of the branch `daves`, and any update that has b
...
-If you do not want to change the git configuration, you can also use:
+If you would also like to set that remote branch as upstream for the current one, use the following instead:
 
 ```sh
-$ git push -u <remote> HEAD
+$ git push -u <remote>
 ```
 
-With the ```upstream``` mode and the ```simple``` mode (default in Git 2.0), the following command will push the current branch w.r.t. the remote branch that has been registered previously with -u :
+With the `upstream` mode and the `simple` (default in Git 2.0) mode of the `push.default` config, the following command will push the current branch w.r.t. the remote branch that has been registered previously with -u :
```
Comment (RichardLitt) [inline README.md]: While we're here, change w.r.t. to 'with regards to', and put ticks around `-u`, and move the colon next to it?

**Comment by eamanu** [comment]
I like the changes.  Please make the requested changed by @RichardLitt

File: README.md (hunk 4 above)
Comment (sivaraam) [reply README.md]: Yep.

Comment (RichardLitt) [reply README.md]: @sivaraam No need to comment - just amend the commit or add a new one and my comment will go away. :)

**Comment by RichardLitt** [comment]
Thank you!

**Pull request merged**